### PR TITLE
Design Changes for Droidcon 2014 b/w Oct 7 - 15th

### PR DESCRIPTION
**Pull request by rajasekharreddy-sagamreddy**

### CODE REVIEW

File: static/css/objects.css [hunk 1]
```diff
@@ -1,18 +1,95 @@
 
+
+.homebanner{
+  background-image: url('https://imgee.s3.amazonaws.com/imgee/802cede3e3d14968a56fdfa26eb2982a_w1296_h830.jpeg');
+  width: 100%;
+  height: auto;
+}
+
+
+
```
Comment (jace) [inline static/css/objects.css]: Don't leave so many blank lines.

File: static/css/objects.css [hunk 2]
```diff
@@ -1,18 +1,95 @@
 
+
+.homebanner{
+  background-image: url('https://imgee.s3.amazonaws.com/imgee/802cede3e3d14968a56fdfa26eb2982a_w1296_h830.jpeg');
```
Comment (jace) [inline static/css/objects.css]: 1296px is too wide for mobile. The default stylesheet is for mobile. You should insert the smaller version of the image here and the larger versions in responsive stylesheets for each breakpoint.

File: static/css/objects.css [hunk 3]
```diff
@@ -1,18 +1,95 @@
...
+    line-height: 0.5;
+    padding: 1em 2em 1em;
+    text-transform: uppercase;
+    text-align: center;
+    border-radius: 0.25em;
+    border: 0;
+    }
+
+.button-Propose-a-session {
+  display: inline-block;
```
Comment (jace) [inline static/css/objects.css]: Use consistent indents.

File: static/css/objects.css [hunk 4]
```diff
@@ -1,18 +1,95 @@
...
+    border-radius: 0.25em;
+    border: 0;
+}
+
+
+
+
+.bcolor{
+
+}
```
Comment (jace) [inline static/css/objects.css]: Empty class?

File: static/css/objects.css [hunk 5]
```diff
@@ -1,18 +1,95 @@
...
+
+.panal {
```
Comment (jace) [inline static/css/objects.css]: Spelling of "panel"

File: static/css/objects.css [hunk 6]
```diff
@@ -1,18 +1,95 @@
...
+  font-size:
```
Comment (jace) [inline static/css/objects.css]: Missing value.

File: static/css/objects.css [hunk 7]
```diff
@@ -1,18 +1,95 @@
...
     text-transform: uppercase;
-    text-align: center;
+    text-align: left;
     border-radius: 0.25em;
     border: 0;
 }
 .button-mini    { padding: 0.2em 0.6em; }
 .button-on-dark { background: #c55a40; }
-.button:focus   { text-decoration: none; }
+.button:focus   { text-decoration: #FFFFFF; }
```
Comment (jace) [inline static/css/objects.css]: `text-decoration` takes `none` or `underline`, not `#FFFFFF`.

File: static/css/theme.css [hunk 8]
```diff
@@ -22,34 +22,69 @@ a.link-silent   { border-bottom: none; }
 
 
 /*  Colours  */
-
+.lightyellow    {color: #F8DC0B !important;}
+.lightcream     {color: #DDD4BC !important;}
+.wordpressblue  {color: #1FDA9A  !important;}
+.creamwhite     {color: #F5EACD !important;}
+.yellowgreen    {color:  #8DC3E9 !important;}
+.liblue         { color: #73C5E1 !important; }
```
Comment (jace) [inline static/css/theme.css]: Fix spacing and indents.

File: static/css/theme.css [hunk 9]
```diff
@@ -22,34 +22,69 @@ a.link-silent   { border-bottom: none; }
...
+
+.fill-lightyellow    {background-color: #F8DC0B !important; color: #FFFFFF; }
+.fill-lightcream     {background-color: #DDD4BC !important; color: #FFFFFF;}
+.fill-wordpressblue  {background-color: #1FDA9A  !important; color: #FFFFFF;}
+.fill-creamwhite     {background-color: #F5EACD !important; color: #FFFFFF;}
+.fill-yellowgreen    {background-color:  #8DC3E9 !important; color: #FFFFFF;}
+.fill-liblue         {background-color: #73C5E1 !important; color: #FFFFFF;}
+.fill-lightblue      {background-color: #FFD966 !important; color: #30302F;}
+.fill-white          {background-color: #FFFFFF !important; color: #30302F;}
+.fill-bwhite         {background-color: #FFFFFF !important; color: #30322F; background-image: url('https://imgee.s3.amazonaws.com/imgee/802cede3e3d14968a56fdfa26eb2982a_w1296_h830.jpeg' ); width:100%;, height: auto;}
```
Comment (jace) [inline static/css/theme.css]: A colour fill class shouldn't have background-image.

File: static/css/typography.css [hunk 10]
```diff
@@ -12,14 +12,14 @@ i,
 em,
 .prose-type     { font-family: "Adelle", serif; }
 */
-
+@import url(http://fonts.googleapis.com/css?family=Varela);
```
Comment (jace) [inline static/css/typography.css:16]: Should have been `Varela Round`, not `Varela`.

File: static/css/typography.css [hunk 11]
```diff
@@ -12,14 +12,14 @@ i,
...
 
 body,
-.title-type     { font-family: "Roboto Slab", brandon-grotesque, "Brandon Grotesque", "Gill Sans", "Gill Sans MT", sans-serif; }
+.title-type     { font-family: "Varela", brandon-grotesque, "Brandon Grotesque", "Gill Sans", "Gill Sans MT", sans-serif; }
 
 i,
 em,
-.prose-type     { font-family: "Roboto Slab", adobe-caslon-pro, "Adobe Caslon Pro", serif; }
+.prose-type     { font-family: "Varela", adobe-caslon-pro, "Adobe Caslon Pro", serif; }
```
Comment (jace) [inline static/css/typography.css]: Remove `brandon-grotesque` and `adobe-caslon-pro`.

File: static/css/typography.css [hunk 12]
```diff
@@ -40,7 +40,8 @@ html                    { font-size: 16px;     line-height: 1.5; }
 .secondary-prose        { font-size: 0.950rem; line-height: 1.40; }
 .tiny                   { font-size: 0.600rem; line-height: 1.60; }
 .tiny-prose             { font-size: 0.500rem; line-height: 1.60; }
-
+.change-font-size       {font-size: 0.890rem; }
+.hack-summ               {font-size: 1.200rem;}
```
Comment (jace) [inline static/css/typography.css]: Both these look like hacks. Where are they being used?

File: templates/data/conference-schedule-day1.html [hunk 13]
```diff
@@ -1,7 +1,7 @@
 {%
   set schedule = {
       'id'    : 'schedule-day1'
-    , 'date'  : '28 Nov'
+    , 'date'  : ' will be updated shortly'
```
Comment (jace) [inline templates/data/conference-schedule-day1.html]: Date is December 5.

File: templates/data/conference-schedule-day2.html [hunk 14]
```diff
@@ -1,28 +1,28 @@
 {%
   set schedule = {
       'id'    : 'schedule-day2'
-    , 'date'  : '29 Nov'
+    , 'date'  : '6th dec'
```
Comment (jace) [inline templates/data/conference-schedule-day2.html]: Remove `th`. In other place above, use `5 Dec`.

File: templates/data/conference-sessions.html [hunk 15]
```diff
@@ -21,32 +21,32 @@
...
+      , 'link'        : ''
+      , 'funnel'      : ''
       , 'preview'     : ''
       , 'featured'    : true
     }
     , 'session-12-30-A' : {
-        'id'          : 'session-12-30-A'
+        'id'          : ''
       , 'title'       : 'Freemium Game Design'
       , 'speaker'     : 'Nilesh Jahagirdar'
```
Comment (jace) [inline templates/data/conference-sessions.html]: Remove these values too.

File: templates/data/conference-venue.html [hunk 16]
```diff
@@ -26,16 +26,16 @@
     , 'directions'  : [
       {
         'from'      : ''
-        , 'route'     : '<p>MLR Convention Centre, Whitefield is just off the main Whitefield Road and is easily accessible from the various nodes of Bangalore.</p>'
+        , 'route'     : '<p>MLR Convention Centre, JP Nagar is easily accessible from the various nodes of Bangalore.</p>'
       }
       , {
-          'from'      : 'BIAL Airport: 46 Kms'
+          'from'      : 'Whitefield: 29 Kms'
```
Comment (jace) [inline templates/data/conference-venue.html]: Include distance from airport instead of Whitefield.

File: templates/data/sponsors.html [hunk 17]
```diff
@@ -57,214 +57,224 @@
...
+      , 'blurb'       : '<p>Helpshift is a industry-leading mobile customer support SaaS provider
+with offices in San Francisco & Pune.We ve built a customizable
```
Comment (jace) [inline templates/data/sponsors.html]: Change `We ve` to `We’ve`.

File: templates/data/sponsors.html [hunk 18]
```diff
@@ -57,214 +57,224 @@
...
+      , 'id'          : ''
+      , 'logo'        : ''
+      , 'link'        : ''
+      , 'type'        : ''
       , 'blurb'       : ''
     }
     , {
-        'title'       : 'Opera'
+        'title'       : ''
       , 'id'          : 'opera'
```
Comment (jace) [inline templates/data/sponsors.html]: Empty this too.

File: templates/layouts/page.html [hunk 19]
```diff
@@ -42,7 +42,7 @@
 {%- if config['DEBUG'] %}
   <link rel="stylesheet" href="{{ theme_static('css/fonts.css') }}">
 {%- else %}
-  <script type="text/javascript" src="//use.typekit.net/ewq6otu.js"></script>
+  <script type="text/javascript"src="//use.typekit.net/ewq6otu.js"></script>
```
Comment (jace) [inline templates/layouts/page.html]: Remove this Typekit section since we are only using Google Fonts.

File: templates/layouts/page.html [hunk 20]
```diff
@@ -59,20 +59,20 @@
...
+            Droidcon India 2014
           </a>
         </h1>
       </div>
       <div class="col-lg-7 col-md-7 col-sm-5 col-xs-5">
-        <h1 class="extrabold legible">Droidcon India</h1>
+        <h1 class="extrabold legible">Droidcon India </h1>
         <h3 class="calm tiny nospace-bottom all-caps">
-          28&nbsp;-&nbsp;30&nbsp;Nov<span class="hidden-xs">ember</span>&nbsp;2013
+          5th&nbsp;-&nbsp;6th&nbsp;<span class="hidden-xs">DECEMBER</span>&nbsp;2014
```
Comment (jace) [inline templates/layouts/page.html]: You have to put "Dec" outside the span and the rest inside, like this:
`5th&nbsp;-&nbsp;6th&nbsp;Dec<span class="hidden-xs">ember</span>&nbsp;2014`

Note that it's not in uppercase.

File: templates/layouts/page.html [hunk 21]
```diff
@@ -80,9 +80,10 @@ <h3 class="calm tiny nospace-bottom all-caps">
 
       <div class="ticket-link-container col-lg-3 col-md-3 col-sm-4 col-xs-4">
         <p class="text-right space-line">
-          <a class="button" href="http://droidcon.doattend.com/">
-            <span class="hidden-xs">Buy Your</span> Ticket<span class="visible-xs">s</span>
+          <a class="button" href="https://in.explara.com/e/droidcon-2014">
+            <span class="hidden-xs"></span>get your Ticket <span class="visible-xs"></span>
```
Comment (jace) [inline templates/layouts/page.html]: "Get your" goes inside the span here.

File: templates/sections/hackathon-summary.html [hunk 22]
```diff
@@ -1,63 +1,27 @@
...
+  <div class="row-fluid container fill-wordpressblue">
+    <h2 class="col-lg-12 col-md-12 col-sm-12 col-xs-12 extralarge all-caps space-one-bottom text-center">What to expect</h2>
+    
+    <div class="row">
+      <div class="col-lg-6  what-to-expect">
+        <ul style="list-style: none;" class="large-prose hack-color">
+          <li class="hack-font ">.</li>
+          <li>TALKS</li>
+          <li>HANDS ON WORK SESSIONS </li>
+          <li>IoT PROJECT DEMO ZONES </li>
```
Comment (jace) [inline templates/sections/hackathon-summary.html]: Don't use uppercase in the text. If you want uppercase in the page, use the CSS rule `text-transform: uppercase`. The source must always be regular case.

File: templates/sections/past-highlights.html [hunk 23]
```diff
@@ -1,44 +1,35 @@
...
-      <h2 class="extralarge all-caps">Previously at<br />Droidcon India</h2>
+      <h6 class="large all-caps">Previously at<br />Droidcon India</h6>
       <ul class="inline-on-mobile list-plain secondary prose">
+        <li><a href="//droidcon.in/2013">Bangalore&nbsp;2013</a></li>
         <li><a href="//droidcon.in/2012">Bangalore&nbsp;2012</a></li>
         <li><a href="//droidcon.in/2011/">Bangalore&nbsp;2011</a></li>
+     
+      <h6 class="large all-caps">Droidcon worldwide <br /></h6>
+      <ul class="inline-on-mobile list-plain secondary prose">
+        <li><a href="http://nyc.droidcon.com/2014/">New York</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://uk.droidcon.com/">London</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://swdc.se/droidcon/events/stockholm-2014/">Stockholm</a></li>
```
Comment (jace) [inline templates/sections/past-highlights.html:12]: Get rid of all the `&nbsp;` elements here. Use CSS to adjust your spacing as needed.

File: templates/sections/past-highlights.html [hunk 24]
```diff
@@ -1,44 +1,35 @@
...
+        
+        <li><a href="http://droidcon.tn/">Tunisia</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://www.androiddeveloperdays.com">Ankara</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://ro.droidcon.com/2014/">Bucharest</a></li>
+        
       </ul>
     </div>
     
     <div class="col-lg-7 col-md-7 col-sm-12 col-xs-12 legible-prose prose prose-type space-one-top">
       <p>
         Droidcon is India's largest Android developer conference, and is part of the world wide series of conferences that happens in <a href="http://uk.droidcon.com/">London</a>, <a href="http://fr.droidcon.com/">Paris</a>, <a href="http://de.droidcon.com/">Berlin</a>, <a href="http://www.droidcon.nl/">Netherlands</a>, <a href="http://droidcon.tn/">Tunis</a>, <a href="http://www.androiddeveloperdays.com/">Ankara</a> and <a href="http://droidcon.com/">Brussels</a>. The first edition of Droidcon India was at Bangalore in Nov 2011. The second edition in Nov 2012 was featuring General &amp; Specialized Topics, Native + HTML5 and App Demos.
+        in 2013, was featuring  dedicated tracks for deep dives into Systems, UX, Gaming, Business & App Demos
```
Comment (jace) [inline templates/sections/past-highlights.html]: Incomplete sentence.

File: static/css/typography.css (hunk 12 above)
Comment (rajasekharreddy-sagamreddy) [inline static/css/typography.css]: in hackathon-summary.html i changed it

**Pull request merged**